### ISSUE TITLE
Is there any plans to release more segmentation models in near future?

### ISSUE THREAD

**Issue by shadyatscu** [dmlc/gluon-cv#775]
Different from other deep learning frameworks, mxnet is very convenient for beginners as the official model zoo has integrated with SOTA algorithms. In the segmentation part, some popular algorithms are missing, eg unet series, linknet and so on... Is there ant plans to add those SOTA algorithms in the near future or any roadmaps?

**Comment by Mut1nyJD** [comment]
I am looking at implementing FastFCN  and ShelfNet I am happy to contribute once I got it working.

**Comment by zhanghang1989** [comment]
DeepLabV3Plus using Xception backbone was just added https://github.com/dmlc/gluon-cv/blob/master/gluoncv/model_zoo/deeplabv3_plus.py#L243
I will provide pretrained model weights soon.

**Comment by OliverColeman** [comment]
While we're waiting for pretrained models (although it looks like there are some for COCO in the code), can you confirm whether you train this in two stages:
1. not dilated with output stride 32, large batch size
2. dilated with output stride 8, small batch size, with batchnorm params fixed?

**Comment by OliverColeman** [comment]
Actually, looks like pretrained model params not actually available yet.
